build/BUILD.bazel

```diff
@@ -22,6 +23,18 @@ licenses(["notice"])  # Apache 2
 
 package(default_visibility = ["//visibility:public"])
 
+bool_flag(
+    name = "enable_remote_tpu",
+    build_setting_default = False,
+)
+
+config_setting(
+    name = "remote_tpu_enabled",
```

Comment:
Maybe rename `legacy_tpu_*`? Up to you

Reply:
I might leave it as is... I think this is a better description of why it's different to the other "TPU".

And hopefully we can remove it sometime anyway!